Detections/MultipleDataSources/AADAWSConsoleCorrelation.yaml

```diff
@@ -37,10 +37,10 @@ query: |
   //See if any of those IPs have sucessfully logged into the AWS console
   AWSCloudTrail
   | where TimeGenerated > ago(1d)
-  | where EventName == "ConsoleLogin"
+  | where EventName =~ "ConsoleLogin"
   | extend LoginResult = tostring(parse_json(ResponseElements).ConsoleLogin) 
-  | where LoginResult == "Success"
-  | where SourceIpAddress in (suspicious_signins)
+  | where LoginResult =~ "Success"
+  | where SourceIpAddress in (Suspicious_signins)
   | extend Reason = "Multiple failed AAD logins from IP address"
   | extend MFAUsed = tostring(parse_json(AdditionalEventData).MFAUsed)
   | extend User = iif(UserIdentityUserName == "", UserIdentityType, UserIdentityUserName) 
```

Comment:
does it make more sense to use isempty(UserIdentityUserName) instead of UserIdentityUserName == ""

Reply:
Done. Thanks